src/components/Sidebar/Sidebar.vue

```diff
@@ -79,16 +90,35 @@
 				/>
 			</SidebarButton>
 		</v-list>
+
+		<div v-if="isMobile" class="py-2" />
+
+		<div
+			v-if="isMobile"
+			class="pr-1 font-weight-light"
+			style="
+				position: absolute;
+				bottom: 0;
+				width: 100%;
+				text-align: center;
+				font-size: 14px;
+				z-index: -1;
+			"
+		>
+			v{{ appVersion }}
+		</div>
```

Comment:
Here, I think we should keep the functionality of clicking the version to open the changelog window on mobile

Reply:
I think the interaction area might be a bit too small. Maybe we can just open the changelog window upon clicking the bridge. logo and remove the version number inside of the sidebar?

Reply:
Yeah thats a good solution